.circleci/config.yml

```diff
@@ -0,0 +1,84 @@
+build-steps: &build-steps
```

Comment:
Overview of build steps on circleCI:
* Build on both 8.10 and 10.16.
* Validate generated JSON with [cfn-lint](https://github.com/aws-cloudformation/cfn-python-lint)
* On `master` branch, deploy to npm and update changelog with [standard-version](https://github.com/conventional-changelog/standard-version)